src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -226,11 +226,14 @@ public static void main(String... args) throws IOException {
         }
         finally {
             // return exit code base on validation of Checker
-            if (errorCounter != 0 && !cliViolations) {
-                final LocalizedMessage errorCounterMessage = new LocalizedMessage(0,
-                        Definitions.CHECKSTYLE_BUNDLE, ERROR_COUNTER,
-                        new String[] {String.valueOf(errorCounter)}, null, Main.class, null);
-                System.out.println(errorCounterMessage.getMessage());
+            // two ifs exist till https://github.com/hcoles/pitest/issues/377
+            if (errorCounter != 0) {
```

Comment:
work-around to calm down pitest till the[ issue on pitest ](https://github.com/hcoles/pitest/issues/377) will be resolved